src/advanced/configuration-updater.md

```diff
@@ -100,10 +101,16 @@ config][stored_configuration] serialization. It has the following fields:
   Hash of the previous active configuration.
 - **actual_from**: integer  
   The height from which the configuration became actual.
-- **validators**: Array<PublicKey\>  
+- **validator_keys**: Array<PublicKey\>  
   List of validators' public keys.
+- **majority_count**: integer  
+  Amount of votes required to commit a new configuration proposal.
+  By default the number of votes is calculated as
+  2/3 + 1 of total validators count.
 - **consensus**: Object  
   Consensus-specific configuration parameters.
+- **consensus.max_message_len**: integer  
```

Comment:
For future-proofness, I suggest to remove all `consensus.*` sub-entries from here and instead link the `consensus` entry to the "[genesis.consensus]" section in the Configuration article.